package/python-kiwi-spec-template

```diff
@@ -231,14 +278,14 @@ outside of the scope of kiwi appliance building.
 %ifarch %ix86 x86_64
 %package -n kiwi-pxeboot
 Summary:        KIWI - PXE boot structure
+License:        GPL-3.0+
+Group:          System/Management
```

Comment:
These tags do not need to be repeated from the main header, as they are inherited during the package build.